src/base_config.js

```diff
@@ -107,6 +107,8 @@ class BaseConfig {
       ...this._genericActions(TYPES.DESTROY),
```

Comment:
Do we still need this, or is what you added below taking its place?

Reply:
We get a couple additional actions with this, and additions below override the necessary actions.

Reply:
oh okay. I figured i was missing something im trying to catch up with whats going on and also review lol